presentation/mySlide.py

```diff
@@ -0,0 +1,94 @@
+import asyncio
+from dataclasses import dataclass
+from typing import List
+import openai
+import time
+import os
+from dotenv import load_dotenv
+
+# before running the code, please make sure that the environment variables section includes:
+# name: OPENAI_API_KEY, value: <value of key>
+# Load environment variables from .env file
+# load_dotenv()
+
+model = 'text-davinci-003'
+max_tokens = 1000
+error_messages = {
+    "rate_limit": "Rate limit exceeded. Waiting for 60 seconds...",
+    "authentication": "Invalid API key.",
+    "exception": "An error occurred while calling the OpenAI API:",
+    "timeout": "Request to OpenAI API timed out."
+}
+
+
+@dataclass
+class MySlide:
+    """
+    Represents a slide in a presentation.
+
+    Attributes:
+        slide_number (int): The slide number.
+        text_boxes (List[str]): List of text boxes in the slide.
+    """
+
+    slide_number: int
```

Comment:
why not in the _init_ of the class, why do you need it a a class attribute and not object one?